### PR TITLE
Docs: Explains roles in the EID system on the home page

### PR DESCRIPTION
**Pull request by lanterno**
Simple explanations for roles in E-ID systems

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,12 @@ and uses Aries Cloud Agent (ACA-Py) as a backend for the issuer and verifier.
 ![platform screenshot](./docs/platform-screenshot-01.png)
 
 ## Technical Details and system requirements
-### Flow Diagram
+### Flow Diagrams
 ![flow diagram](./docs/eid-flow-diagram.jpg)
+
+**E-ID issuance flow**
+![flow diagram](./docs/issuance-diagram.png)
```
Comment (ineiti) [inline README.md:13]: Please add a reference to the original file here.

File: pages/index.vue [hunk 2]
```diff
@@ -13,7 +13,7 @@
               <div class="absolute left-0 top-0 flex h-10 w-10 items-center justify-center rounded-lg bg-indigo-600">
                 <component :is="feature.icon" class="h-6 w-6 text-white" aria-hidden="true" />
               </div>
-              {{ feature.name }}
+              <a :href="feature.url">{{ feature.name }}</a>
```
Comment (ineiti) [inline pages/index.vue:16]: curiosity: what is the `:href` doing?

File: pages/index.vue [hunk 3]
```diff
@@ -30,26 +30,35 @@ const features = [
...
+        "Entity that issues the credentials. In our example, the issuer will be the ACME school." +
+        "The issuer will make sure as well to sync all public keys and schemas with the ledger.",
     icon: CloudArrowUpIcon,
+    url: "/issuer"
   },
   {
     name: 'Ledger',
     description:
-        'Write a description here...',
+        "Ledgers are also called 'trust anchor'. It is the reference used by verifiers to " +
```
Comment (ineiti) [inline pages/index.vue:41]:
```suggestion
        "A ledger is the 'trust anchor' of the system. It is the reference used by verifiers to " +
```

File: pages/index.vue [hunk 4]
```diff
@@ -30,26 +30,35 @@ const features = [
...
+        "verify the authenticity of the credentials. That is done by storing all public keys of all" +
```
Comment (ineiti) [inline pages/index.vue:42]:
```suggestion
        "verify the signatures of the credentials. The ledger stores all public keys of all" +
```

File: pages/index.vue [hunk 5]
```diff
@@ -30,26 +30,35 @@ const features = [
...
+        "Issuers on the ledger in addition to the schemas and revocation registries",
```
Comment (ineiti) [inline pages/index.vue:43]:
```suggestion
        "issuers, as well as the schemas and revocation registries.",
```

File: pages/index.vue [hunk 6]
```diff
@@ -30,26 +30,35 @@ const features = [
...
     icon: LockClosedIcon,
+    url: "https://explorer.sandbox.ssi.ch/home/SANDBOX"
   },
   {
     name: 'Wallet',
     description:
-        'Write a description here...',
+        "Storage application that is the only place that will store the actual credentials." +
+        "The wallet is also be the only place that can present the credentials to a verifier." +
+        "In our example, the wallet will be owned by the student.",
```
Comment (ineiti) [inline pages/index.vue:52]:
```suggestion
        "The wallet is an application which stores the credentials of a user." +
        "It also prepares the credentials and presents them to the verifier." +
        "In our example, the wallet holds the diplomas of the student.",
```

File: pages/index.vue [hunk 7]
```diff
@@ -30,26 +30,35 @@ const features = [
...
     icon: ArrowPathIcon,
+    url: "https://github.com/e-id-admin/public-sandbox-trustinfrastructure/discussions/17"
   },
   {
     name: 'Verifier',
     description:
-        'Write a description here...',
+        "Entity that needs to verify the credentials. In our example, the verifier will be the Leo Inc.",
```
Comment (ineiti) [inline pages/index.vue:59]:
```suggestion
        "Entity that can verify credentials. In our example, the verifier is Leo Inc.",
```

File: pages/index.vue [hunk 8]
```diff
@@ -30,26 +30,35 @@ const features = [
   {
     name: 'Issuer',
     description:
-        'Write a description here...',
+        "Entity that issues the credentials. In our example, the issuer will be the ACME school." +
+        "The issuer will make sure as well to sync all public keys and schemas with the ledger.",
```
Comment (ineiti) [inline pages/index.vue:34]:
```suggestion
        "Entity that issues the credentials by signing them. In our example, the issuer is the ACME school." +
        "The public key and the schemas used by the issuer are stored on the ledger.",
```

**Review by ineiti — CHANGES_REQUESTED** [review]

File: pages/index.vue (hunk 2 above)
Comment (lanterno) [reply pages/index.vue:16]: the `:` is Vue's way of saying "bind this parameter to the following value" 

so this binds the href parameter to the variable in `feature.url`

File: pages/index.vue [hunk 9]
```diff
@@ -30,33 +30,33 @@ const features = [
...
+        "It also prepares the credentials and presents them to the verifier." +
+        "In our example, the wallet holds the diplomas of the student.",
     icon: ArrowPathIcon,
     url: "https://github.com/e-id-admin/public-sandbox-trustinfrastructure/discussions/17"
   },
   {
     name: 'Verifier',
     description:
-        "Entity that needs to verify the credentials. In our example, the verifier will be the Leo Inc.",
+        "Entity that can verify credentials. In our example, the verifier is Leo Inc.",
```
Comment (PascalinDe) [inline pages/index.vue:59]:
```suggestion
        "Entity that verifies credentials. In our example, the verifier is Leo Inc.",
```

File: pages/index.vue [hunk 10]
```diff
@@ -30,33 +30,33 @@ const features = [
...
     icon: LockClosedIcon,
     url: "https://explorer.sandbox.ssi.ch/home/SANDBOX"
   },
   {
     name: 'Wallet',
     description:
-        "Storage application that is the only place that will store the actual credentials." +
-        "The wallet is also be the only place that can present the credentials to a verifier." +
-        "In our example, the wallet will be owned by the student.",
+        "The wallet is an application which stores the credentials of a user." +
```
Comment (PascalinDe) [inline pages/index.vue:50]:
```suggestion
        "The wallet is an application installed by the user which stores their credentials." +
```

File: pages/index.vue [hunk 11]
```diff
@@ -30,33 +30,33 @@ const features = [
...
   },
   {
     name: 'Ledger',
     description:
-        "Ledgers are also called 'trust anchor'. It is the reference used by verifiers to " +
-        "verify the authenticity of the credentials. That is done by storing all public keys of all" +
-        "Issuers on the ledger in addition to the schemas and revocation registries",
+        "A ledger is the 'trust anchor' of the system. It is the reference used by verifiers to " +
+        "verify the signatures of the credentials. The ledger stores all public keys of all" +
+        "issuers, as well as the schemas and revocation registries",
```
Comment (PascalinDe) [inline pages/index.vue:43]:
```suggestion
        "A ledger is the 'trust anchor' of the system. It stores the public keys of the" +
        "issuers, as well as the schemas and revocation registries. It is the reference used by verifiers to " +
        "verify the signatures of the credentials.",
```

File: pages/index.vue (hunk 2 above)
Comment (ineiti) [reply pages/index.vue:16]: > the `:` is Vue's way of saying "bind this parameter to the following value"

TIL... - so this is equivalent of

```
<a href="{{ feature.url }}">...
```
?

I'm also surprised by the `"` around `feature.url` - but as I said, I don't know vue.js...

**Comment by ineiti** [comment]
I like PR discussions about text :)

**Review by ineiti — APPROVED** [review]

**Pull request merged**